che-dev.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Looks like this file is unused since `Devfile` references `.che-dev.yaml`

Reply:
it is suppose to be renamed to .che-dev.yaml